lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -194,7 +201,10 @@ def _export_vhd(self, platform: AzurePlatform, virtual_machine: Any) -> str:
             resource_group_name=runbook.shared_resource_group_name,
         )
 
-        path = _generate_vhd_path(container_client, runbook.file_name_part)
+        if not runbook.custom_blob_full_name:
+            path = _generate_vhd_path(container_client, runbook.file_name_part)
+        else:
+            path = runbook.custom_blob_full_name
```

Comment:
there is a possibility that this blob storage account is different with the one used to construct container_client, then copy will fail

Reply:
The custom_blob_full_name doesn't contain the storage account and container name. It's just the blob name. Let me change it to "custom_blob_name".